client/components/chart/bar.jsx

```diff
@@ -137,26 +137,22 @@ module.exports = React.createClass( {
 	render: function() {
 		debug( 'Rendering bar', this.state );
 
-		var barStyle,
-			barClass,
-			count = this.props.count || 1;
-
-		barClass = { chart__bar: true };
+		const barClass = { chart__bar: true };
+		const count = this.props.count || 1;
+		const barStyle = {
+			width: ( ( 1 / count ) * 100 ) + '%'
+		};
 
-		if ( this.props.className ){
+		if ( this.props.className ) {
 			barClass[ this.props.className ] = true;
 		}
 
-		barStyle = {
-			width: ( ( 1 / count ) * 100 ) + '%'
-		};
-
 		return (
 			<div onClick={ this.clickHandler }
-				 onMouseEnter={ this.mouseEnter }
-				 onMouseLeave={ this.mouseLeave }
-				 className={ classNames( barClass ) }
-				 style={ barStyle }>
+				onMouseEnter={ this.mouseEnter }
+				onMouseLeave={ this.mouseLeave }
+				className={ classNames( barClass ) }
```

Comment:
Looks like this could just be `className="chart__bar"` and line 140 can be removed.